ui/packages/catalog-ui-search/src/main/webapp/react-component/user-settings/user-settings.tsx

```diff
@@ -72,14 +73,14 @@ const Header = styled.div`
   padding: 0px ${props => props.theme.mediumSpacing};
 `
 
-const NavigationButton = styled(Button)`
+export const NavigationButton = styled(Button)`
```

Comment:
Doesn't this have to be `styled(Button as any)` now that the styled update got in?